FWCore/Framework/src/ScheduleItems.cc

```diff
@@ -36,6 +38,7 @@ namespace edm {
       preg_(new SignallingProductRegistry(preg)),
       branchIDListHelper_(new BranchIDListHelper),
       thinnedAssociationsHelper_(new ThinnedAssociationsHelper),
+      subProcessParentageHelper_(new SubProcessParentageHelper),
```

Comment:
How about switching to `std::make_shared<>()`? That is actually more memory efficient